tests/pipelines/kit/upstream-load.yaml

```diff
@@ -1,120 +0,0 @@
----
```

Comment:
what is the reason for deleting this file ?

Reply:
So perviously we have pipeline for eksctl, this one and the new pipeline from this PR. They are separated into eksctl, kit-cluster, and kitctl folders. After some discussion we think it will be cleaning to just have one for eksctl and one for kitctl. 
I have revised it back to the three folder structure to keep all of these pipelines.